source/integrations.html.erb

```diff
@@ -0,0 +1,225 @@
+        <a class="btn btn-primary" href="/extensions#extensions-group-shipping" target="_blank">View extensions</a>
+      </div>
+      <div class="integration__list col-md">
+        <a href="https://github.com/solidusio-contrib/solidus_shipwire" class="integration__list__item" target="_blank">
+          <div class="integration__list__item__image">
+            <img class="img-fluid" src="<%= image_path('integrations/shipwire.png') %>" width="90" height="auto">
+          </div>
+          <div class="integration__list__item__description">
+            <h4>Shipwire</h4>
+            <p>Solidus and Shipwire connect manager.</p>
+          </div>
+        </a>
+        <a href="https://github.com/boomerdigital/solidus_shipstation" class="integration__list__item" target="_blank">
+          <div class="integration__list__item__image">
+            <%= inline_svg("integrations/shipstation.svg", class: "isvg") %>
+          </div>
+          <div class="integration__list__item__description">
+            <h4>ShipStation</h4>
+            <p>Enables ShipStation to pull shipments from the system and update tracking numbers.</p>
+          </div>
+        </a>
+        <a href="https://github.com/solidusio-contrib/solidus_easypost" class="integration__list__item" target="_blank">
+          <div class="integration__list__item__image">
+            <%= inline_svg("integrations/easypost.svg", class: "isvg") %>
+          </div>
+          <div class="integration__list__item__description">
+            <h4>EasyPost</h4>
+            <p>Solidus integration with EasyPost.</p>
+          </div>
+        </a>
+      </div>
+    </div>
+  </div>
+</div>
+
+<div class="integration content-block" id="marketing">
+  <div class="container">
+    <div class="row align-items-center">
+      <div class="col-md">
+        <%= inline_svg("icons/marketing.svg", class: "isvg group") %>
+        <h2>Marketing</h2>
+        <p>Collecting data and using it in meaningful ways can be challenging. Marketing and analytics tools help you generate useful insights that inform your business decisions and promotional campaigns.</p>
+        <p>Solidus has integrations for the best analytics and marketing tools, as well as a powerful event-based architecture to easily integrate new services.</p>
+        <a class="btn btn-primary" href="/extensions#extensions-group-marketing" target="_blank">View extensions</a>
+      </div>
+      <div class="integration__list col-md offset-md-1">
+        <div class="integration__list__item">
+          <div class="integration__list__item__image">
+            <%= inline_svg("integrations/google_analytics.svg", class: "isvg") %>
+          </div>
+          <div class="integration__list__item__description">
+            <h4>Google Analytics</h4>
+            <p>Easily integrate Solidus with Google Analytics.</p>
```

Comment:
I think that having this not linking anyway is a blocking. We should at least have some resources about how to integrate GA in a Solidus store if we want to keep this link here.

Reply:
The GA card is not a link though, but I agree it should be linked to a resource or an extension as the other services. Do we have a tutorial to share? Or do we have to create it?

Reply:
It's new content, unfortunately we don't have it yet.

Reply:
Hi 👋 
I meddle in this conversation: we could add a tag like "Upcoming Integration", or something similar. Sounds good?